integration-test/src/main/java/com/sequenceiq/it/cloudbreak/util/ssh/action/SshJClientActions.java

```
@@ -257,6 +257,24 @@ private void checkNoOutboundInternetTraffic(String instanceIp) {
         }
     }
 
+    public SdxTestDto checkKinitDuringFreeipaUpgrade(SdxTestDto testDto, List<InstanceGroupV4Response> instanceGroups, List<String> hostGroupNames) {
+        getSdxInstanceGroupIps(instanceGroups, hostGroupNames, false).stream().findFirst().ifPresent(ip -> checkKinitDuringFreeipaUpgrade(ip));
```

if no instance is found it should throw an error as this test won't run